src/main/java/com/google/dicomwebfuse/fuse/DicomFuse.java

```diff
@@ -105,8 +105,12 @@ public int opendir(String path, FuseFileInfo fi) {
   public int read(String path, Pointer buf, @size_t long size, @off_t long offset,
       FuseFileInfo fi) {
     try {
-      DicomPath dicomPath = dicomPathParser.parsePath(path);
-      return dicomFuseHelper.readInstance(dicomPath, buf, (int) size, offset);
+      if (parameters.getCacheTime().getInstanceFilesCacheTime() > 0) {
```

Comment:
Can we add a unit test for this change?

Reply:
Done

Reply:
The new unit tests depend on the libfuse2 library and broke the previous integration test. cloudbuild.yaml and build.sh were updated to pass new unit tests.